cmd/promu.go

```diff
@@ -156,3 +156,12 @@ func envOr(name, def string) string {
 	}
 	return s
 }
+
+func checkRequiredConfigurations(configVars []string) {
+	for _, configVar := range configVars {
+		if !viper.IsSet(configVar) {
+			err := fmt.Errorf("'%s'", configVar)
+			fatalMsg(err, "Missing required configuration")
```

Comment:
I've change the verification function to return a `bool` and a `error`. The `fatal()` is handled outside of the function.